samples/sensor/bme68x_iaq/sample.yaml

```diff
@@ -35,3 +35,11 @@ tests:
       ordered: true
       regex:
         - "(\\d+\\.\\d+); press: (\\d+\\.\\d+); humidity: (\\d+\\.\\d+); iaq: (\\d+)"
+  sample.sensor.bme68x.spi:
+    depends_on: spi
+    harness: sensor
+    integration_platforms:
```

Comment:
looks like we have to remove the `integration_platforms` part for the new test - my fault